modules/tinymce/src/themes/silver/main/ts/ui/general/UiFactory.ts

```diff
@@ -34,54 +35,57 @@ import { renderHtmlPanel } from './HtmlPanel';
+  textarea: make<Dialog.TextArea>((spec, backstage) => renderTextarea(spec, backstage.shared.providers)),
+  label: make<Dialog.Label>((spec, backstage) => renderLabel(spec, backstage.shared)),
   iframe: makeIframe((spec, backstage) => renderIFrame(spec, backstage.shared.providers)),
-  button: make((spec, backstage) => renderDialogButton(spec, backstage.shared.providers)),
-  checkbox: make((spec, backstage) => renderCheckbox(spec, backstage.shared.providers)),
-  colorinput: make((spec, backstage) => renderColorInput(spec, backstage.shared, backstage.colorinput)),
-  colorpicker: make(renderColorPicker), // Not sure if this needs name.
-  dropzone: make((spec, backstage) => renderDropZone(spec, backstage.shared.providers)),
-  grid: make((spec, backstage) => renderGrid(spec, backstage.shared)),
-  listbox: make((spec, backstage) => renderListBox(spec, backstage)),
-  selectbox: make((spec, backstage) => renderSelectBox(spec, backstage.shared.providers)),
-  sizeinput: make((spec, backstage) => renderSizeInput(spec, backstage.shared.providers)),
-  slider: make((spec, backstage) => renderSlider(spec, backstage.shared.providers)),
-  urlinput: make((spec, backstage) => renderUrlInput(spec, backstage, backstage.urlinput)),
-  customeditor: make(renderCustomEditor),
-  htmlpanel: make(renderHtmlPanel),
-  imagetools: make((spec, backstage) => renderImageTools(spec, backstage.shared.providers)),
-  table: make((spec, backstage) => renderTable(spec, backstage.shared.providers)),
-  panel: make((spec, backstage) => renderPanel(spec, backstage))
+  button: make<Dialog.Button>((spec, backstage) => renderDialogButton(spec, backstage.shared.providers)),
+  checkbox: make<Dialog.Checkbox>((spec, backstage) => renderCheckbox(spec, backstage.shared.providers)),
+  colorinput: make<Dialog.ColorInput>((spec, backstage) => renderColorInput(spec, backstage.shared, backstage.colorinput)),
+  colorpicker: make<Dialog.ColorPicker>(renderColorPicker), // Not sure if this needs name.
+  dropzone: make<Dialog.DropZone>((spec, backstage) => renderDropZone(spec, backstage.shared.providers)),
+  grid: make<Dialog.Grid>((spec, backstage) => renderGrid(spec, backstage.shared)),
+  listbox: make<Dialog.ListBox>((spec, backstage) => renderListBox(spec, backstage)),
+  selectbox: make<Dialog.SelectBox>((spec, backstage) => renderSelectBox(spec, backstage.shared.providers)),
+  sizeinput: make<Dialog.SizeInput>((spec, backstage) => renderSizeInput(spec, backstage.shared.providers)),
+  slider: make<Dialog.Slider>((spec, backstage) => renderSlider(spec, backstage.shared.providers)),
+  urlinput: make<Dialog.UrlInput>((spec, backstage) => renderUrlInput(spec, backstage, backstage.urlinput)),
+  customeditor: make<Dialog.CustomEditor>(renderCustomEditor),
+  htmlpanel: make<Dialog.HtmlPanel>(renderHtmlPanel),
+  imagetools: make<Dialog.ImageTools>((spec, backstage) => renderImageTools(spec, backstage.shared.providers)),
```

Comment:
This will conflict #7467, so one of us will have some work when the other merges first 😂